### ISSUE TITLE
Tarzan

### ISSUE THREAD

**Issue by aymericingargiola** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#33]
# Game summary
- Game name: Tarzan
- Game version: EU

# Daedalus X64 summary
- Version: v0.1

# Issues
Game boots but menu are zoomed and out of screen. Ingame it's playable but at 13fps.

# Screenshots
![Tarzan](https://i.ibb.co/89gsFYL/2020-05-01-193302.jpg)

# Recommended labels
Ingame (high)

**Comment by Aymen545** [comment]
same goes for the GER (german) FR (france) and US versions

**Comment by VRtinker** [comment]
This no longer works on any version past v1.18 (according to the credits). It will load a black screen @60fps, no audio, no crashes, just no game. Definitely a regression somewhere

**Comment by Rinnegatamante** [comment]
Does this happen with audio off too? (Tarzan uses an audio ucode we still don't have).
Also what about dynarec off?

**Comment by VRtinker** [comment]
Yes, no matter what settings permutation I try, it's always resulting in a black screen

**Comment by aymericingargiola** [comment]
Yep blackscreen only, don't boot atm.

**Comment by Rinnegatamante** [comment]
Latest nightly has this solved, if you can re-test it and report again info.

**Comment by Dreamer1972** [comment]
Using Dx64 nightly (ba53a2)

Legacy rend...Menus look good. Runs fast.
Modern rend - menu text is fine. Black boxes behind some text.
Runs at 60fps.  Too fast.  Use frame limiter for normal speed (30fps).  No audio.  
Very smooth....gameplay is 'perfect'.
